models/models.go

```diff
@@ -104,6 +104,19 @@ type StepIDData struct {
 	Version string
 }
 
+func (stepIDData StepIDData) isLatest() bool {
+	return len(stepIDData.Version) == 0
+}
+
+func (stepIDData StepIDData) validate(defaultStepLibSource string) (err error) {
+	if !stepIDData.isLatest() &&
```

Comment:
I recommend to inline the `isLatest` function.

From the perspective of the `validate` function, the rule is not to "validate version string if not latest", but to validate it "if not empty".

There is no added value in the context of the `validate` function if we know, that an empty version string resolves to the latest version.